### PR TITLE
feat: remove migration for P9130+P9135

### LINKED ISSUE #1146: Remove and rename the migrations
### Context
For Litentry/Rococo: the upgrade was carried out successfully.

---
:heavy_check_mark: Please set appropriate **labels** and **assignees** if applicable.

### PR DESCRIPTION
**Pull request by wangminqi**
This closes #1146

### CODE REVIEW

File: runtime/litentry/src/migration/migration.md [hunk 1]
```diff
@@ -3,4 +3,11 @@ https://github.com/litentry/litentry-parachain/releases/tag/v0.9.10
  # This migration is for the replacement of CollatorSelection with ParachainStaking
 
  <!-- MigrateCollatorSelectionIntoParachainStaking -->
- # The main purpose of runtime upgrade is for make up the missing genesis build of ParachainStaking and clean the old CollatorSelection storage.
\ No newline at end of file
+ # The main purpose of runtime upgrade is for make up the missing genesis build of ParachainStaking and clean the old CollatorSelection storage.
+
+ P9130.rs
```
Comment (Kailai-Wang) [inline runtime/litentry/src/migration/migration.md]: I think it should be `9135`.

The format is a bit off too: https://github.com/litentry/litentry-parachain/blob/c0191d4f024a1d6d7edc2d8d316669dc7c20f700/runtime/litentry/src/migration/migration.md

File: runtime/rococo/src/lib.rs [hunk 2]
```diff
@@ -81,9 +81,6 @@ include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 
 pub mod asset_config;
 pub mod constants;
-// rococo already upgraded to runtime 9130
-// TODO: delete the commented lines and rename the migration file
-// pub mod migration;
```
Comment (Kailai-Wang) [inline runtime/rococo/src/lib.rs:86]: line `// migration::MigrateAtStakeAutoCompound<Runtime>,` needs to be removed too

**Review by Kailai-Wang — APPROVED** [review]
Approved, but some small problems

File: runtime/litentry/src/migration/migration.md (hunk 1 above)
Comment (wangminqi) [reply runtime/litentry/src/migration/migration.md]: Adjusted!

**Comment by Kailai-Wang** [comment]
Looks good, thanks

**Review by Kailai-Wang — APPROVED** [review]

**Pull request merged**